fedimint-client/src/module/mod.rs

```diff
@@ -283,6 +283,26 @@ where
         self.client.get().has_active_states(op_id).await
     }
 
+    pub async fn get_own_active_states(&self) -> Vec<(M::States, ActiveState)> {
+        self.client
+            .get()
+            .executor
+            .get_active_states()
+            .await
+            .into_iter()
+            .filter(|s| s.0.module_instance_id() == self.module_instance_id())
+            .map(|s| {
+                (
+                    s.0.as_any()
+                        .downcast_ref::<M::States>()
+                        .expect("incorrect output type passed to module plugin")
+                        .clone(),
+                    s.1,
+                )
+            })
+            .collect()
+    }
+
```

Comment:
Seems useful, but did you want to apply this fn somewhere? Just want to make sure you didn't forget.

Reply:
I did, but then I need to get something else first to use it, and then got to database ...

I will use it soon.